README.md

```diff
@@ -40,7 +40,7 @@ git clone https://github.com/TOSIT-IO/tdp-getting-started.git
 # Move into project dir
 cd tdp-getting-started
 # Setup local env with stable tdp-collection (mandatory), tdp-lib (mandatory), tdp-server, tdp-collection-extras, tdp-collection-prerequisites, and vagrant
-./scripts/setup.sh -e server -e extras -e prerequisites -e vagrant
+./scripts/setup.sh -e observability -e server -e extras -e prerequisites -e vagrant
```

Comment:
The order is important for collection, `observability` should be after `extras` because it will observe service is extra too no ?

Reply:
Since it's only about the checkout of the submodules in the case of the collections does it really matter?

Reply:
It matter when the script write the `TDP_COLLECTION_PATH` variable inside `.env` file.

Reply:
Re ordered in the latest push force

Reply:
Reordered in the latest push force